tests/cross_compilation/config.toml

```diff
@@ -0,0 +1,10 @@
+[core_settings]
+# Using the default options for the core settings as Parsec will not be run for the
+# cross-compilation test.
+
+[listener]
+listener_type = "DomainSocket"
+timeout = 200 # in milliseconds
+
+# No providers are compiled in for the cross-compilation example.
```

Comment:
Not sure this is the case - the providers are compiled, this config file is just for actually running the service, right?

Reply:
Well as of now, the cross-compilation test only compiles the service for Aarch64 and does not even execute any test, so this file is useless. I will remove it.